modules/preferred-installation-requirements-ibm-z.adoc

```diff
@@ -7,16 +7,17 @@
 // = Minimum resource requirements for cluster installation
 // Do a regular check for changes in modules/installation-minimum-resource-requirements.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: REFERENCE
 [id="preferred-ibm-z-system-requirements_{context}"]
 = Preferred {ibm-z-title} system environment
 
 The preferred system environment for running {product-title} version {product-version} on {ibm-z-name} hardware is as follows:
 
 [discrete]
+[id="preferred-ibm-z-hardware-requirements_{context}"]
 == Hardware requirements
 
-* Three LPARS that each have the equivalent of six IFLs, which are SMT2 enabled, for each cluster.
+* Three logical partitions (LPARs) that each have the equivalent of six Integrated Facilities for Linux (IFLs), which are SMT2 enabled, for each cluster.
```

Comment:
```suggestion
* Three logical partitions (LPARs). For each cluster, every LPAR must have the equivalent of 6 Integrated Facilities for Linux (IFLs) that are SMT2 enabled.
```
More clear broken up into two, I think?

Reply:
it's a lot to hold in your head while you read the sentence (I have many such heavily-modified phrases in my own content areas, I get it :sweat_smile:). If there _is_ a way to preserve the meaning while splitting it into a couple separate sentences, it might help the reader.